build.gradle

```diff
@@ -79,8 +79,8 @@ dependencies {
   api localGroovy()
   api "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
 
-  implementation "org.jetbrains.dokka:dokka-android-gradle-plugin:$dokkaVersion"
-  implementation "org.jetbrains.dokka:dokka-gradle-plugin:$dokkaVersion"
+  implementation "org.jetbrains.dokka:dokka-gradle-plugin:0.10.0"
```

Comment:
Why did we get rid of the `dokkaVersion` variable?

Reply:
Wasn't intentional.

Reply:
fb997e6